index.md

```diff
@@ -1,5 +1,63 @@
-# Welcome to VideOut!
+# Welcome to Videout!
 
-VideOut is a language design to edit videos/gifs/photos through coding. This way programmers that have
+Videout is a language design to edit *videos/gifs* through coding. This way programmers that have
 no experience with media editing programs, can do projects that involve said task. The language was 
-designed as an asignment for the Programming Langages course at UPRM. 
+designed as an assignment for the Programming Language course at UPRM. 
+
+The video editing tools that this language allows you to use are: 
+* Crop
```

Comment:
This should be under a `Basic Language Syntax and Operations` or `Features` header
and include example code as you have done below

Reply:
I picked Basic Language Syntax and Operations since it seems more specific than features.